src/topp/CometAdapter.cpp

```diff
@@ -429,15 +429,21 @@ class TOPPCometAdapter :
     // high res ms/ms:    0.02 tolerance, 0.0 offset (mono masses), theoretical_fragment_ions = 0
 
     String instrument = getStringOption_("instrument");
-    double bin_tol = getDoubleOption_("fragment_bin_tolerance");
+    double bin_tol = getDoubleOption_("fragment_mass_tolerance") * 2; // convert 1-sided tolerance to bin size
```

Comment:
is this an ad-hoc solution?

Reply:
no, just to make the parameters between engines more comparable.